R/module-template.R

```diff
@@ -307,8 +307,9 @@ doEvent.", name, " = function(sim, eventTime, eventType, debug = FALSE) {
 ### template for your event1
 ", name, "Event1 <- function(sim) {
   # ! ----- EDIT BELOW ----- ! #
-
-
+  # THE BELOW TWO LINES ARE FOR INITIALATING UNIT TESTS, DELETE THEM WHEN YOU COMPILE YOUR OWN EVENT
```

Comment:
discussing package tests in this template is likely confusing to the user. we can simply put dummy values as you've done with a note that the user can change /remove them. likewise below.